cmd/relay/events/pebblepersist.go

```diff
@@ -0,0 +1,262 @@
+package events
```

Comment:
could yank this file if not using pebble persister?

Reply:
result of vendoring events/...

cmd/relay/events/consumer.go has #sync which should go upstream
cmd/relay/events/events.go has #sync which should go upstream

diskpersist.go has real changes; changes in db model; time-ish seq

But yes, I can un-vendor a couple types I didn't use.